app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/CoinItem.kt

```diff
@@ -130,13 +132,14 @@ fun CoinItemPreview() {
 
 @Suppress("FunctionNaming")
 @Composable
-@Preview
+@Preview(uiMode = UI_MODE_NIGHT_YES)
 fun CoinItemPreviewDark() {
-    ComposeTheme(darkTheme = true) {
+    ComposeTheme {
         CoinItem(coinItemPreview)
     }
 }
 
+@Suppress("MagicNumber")
 internal val coinItemPreview = CoinItemUiModel(
```

Comment:
@luongvo, Regarding [this](https://github.com/nimblehq/jetpack-compose-crypto/pull/27#discussion_r963409790), What about if we rename `coinItemPreview` to `coinItemUiModelPreview` and move to `CoinItemUiModel` class. 🤔

Reply:
@Wadeewee @hoangnguyen92dn As discussed, I use `PreviewProvider` to optimize it ✅ Fixed in https://github.com/nimblehq/jetpack-compose-crypto/pull/31/commits/15c4c3a7aa2b58c24abf384bf4fe46dfecf39ebe